### PR TITLE
add otel in focus july

### PR DESCRIPTION
**Pull request by austinlparker**
n/t

---

**Preview**: https://deploy-preview-3129--opentelemetry.netlify.app/blog/2023/otel-in-focus-07/

### CODE REVIEW

**Review by cartermp — COMMENTED** [review]
LGTM modulo the textlint stuff (that I disagree with, hmmm, lemme se ehow to change that)

**Comment by austinlparker** [comment]
what's weird is that npm fix stuff isn't showing i have any errors in this file...

**Comment by cartermp** [comment]
Doin' here: https://github.com/open-telemetry/opentelemetry.io/pull/3130

File: content/en/blog/2023/otel-in-focus-07.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+specification. This release is a substantial one, with several notable breaking
+changes to `messaging`, `http`, and `net` namespaces. Please carefully review
+the release notes for more details.
+
+##### [Collector](/docs/collector/)
+
+[Version 0.82.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.82.0)
+has been released, along with 0.81.
+
+The major addition in 0.82 is the ability to export Collector telemetry via
```
Comment (svrnm) [inline content/en/blog/2023/otel-in-focus-07.md:42]:
```suggestion
The major addition in 0.82 is the ability to export collector telemetry via
```

File: content/en/blog/2023/otel-in-focus-07.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+includes a variety of updates and bug fixes.
+
+[Java Instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.28.0)
+has also been released, with some required migrations due to semantic convention
+updates.
+
+#### [Python](/docs/instrumentation/python/)
+
+[Version 1.19](https://github.com/open-telemetry/opentelemetry-python/releases/tag/v1.19.0)
+includes updates to OTLP and Log support.
```
Comment (svrnm) [inline content/en/blog/2023/otel-in-focus-07.md:66]:
```suggestion
includes updates to OTLP and log support.
```

File: content/en/blog/2023/otel-in-focus-07.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+Here are the latest updates from some of our core repositories.
+
+<!-- markdownlint-disable heading-increment -->
+
+##### [Specification](/docs/specs/otel/)
+
+[Version 1.23](https://github.com/open-telemetry/opentelemetry-specification/releases/tag/v1.23.0)
+does not include any significant changes, but some refinements have been made to
+the configuration of TracerProvider and MeterProvider.
```
Comment (svrnm) [inline content/en/blog/2023/otel-in-focus-07.md:27]:
```suggestion
the configuration of  `TracerProvider` and `MeterProvider`.
```

**Review by svrnm — COMMENTED** [review]
some minor suggestions

**Review by chalin — APPROVED** [review]
A few minor copyedits, otherwise LGTM once the textliner is happy.

Maybe for now we can just use "bugfix" and fix the rule after?

File: content/en/blog/2023/otel-in-focus-07.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+<!-- markdownlint-disable heading-increment -->
+
+##### [Specification](/docs/specs/otel/)
+
+[Version 1.23](https://github.com/open-telemetry/opentelemetry-specification/releases/tag/v1.23.0)
+does not include any significant changes, but some refinements have been made to
+the configuration of `TracerProvider` and `MeterProvider`.
+
+##### [Semantic Conventions](/docs/specs/semantic-conventions/)
```
Comment (chalin) [inline content/en/blog/2023/otel-in-focus-07.md:29]:
```suggestion
##### [Semantic Conventions](/docs/specs/semconv/)
```

**Review by chalin — APPROVED** [review]
Link fix is needed, otherwise, LGTM

**Comment by chalin** [comment]
Woohoo, all checks are green!
Any other comments @svrnm @cartermp. If not, let's get this merged!?!

**Review by svrnm — APPROVED** [review]

**Pull request merged**